### ISSUE TITLE
Arrows along a Path ?

### ISSUE THREAD

**Issue by maajdl** [ioam/holoviews#1925]
I could not find a "reasonable" way of  adding arrows along the segments of a Path.
The aim is to indicate directions of a process on a x-y diagram.
I considered a Vector Field, but this is not really a good solution.
The Arrow element doesn't seem suitable for this purpose.

Any suggestion?
Thanks,
Michel

**Comment by jlstevens** [comment]
Makes sense as a use case. I don't have anything specific to recommend right now but we will think about a solution.
